### PR TITLE
#205  Rebase backup system on rsync

### LINKED ISSUE #205: Makefile:67-68: Resurrect backup system. Right now it's...
The puzzle `142-0169d767` from #142 has to be resolved:

https://github.com/fidals/stroyprombeton/blob/89cc959e8665f358e437b7d2b0b35a76d2e4f1d4/docker/Makefile#L67-L68

The puzzle was created by Artemiy on 15-Jun-18. 

Estimate: 60 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by duker33**
Closes #205

### CODE REVIEW

File: etc/backup/backup-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/env sh
+
+# @todo #205:30m Enable rsync --backup option.
+#  To leave changed and/or deleted media files saved.
+
+# sync local media with remote server, that hosts backups
```
[the remark this answers is not in the document]
Comment (artemiy312) [reply etc/backup/backup-entrypoint.sh]: It would be nice to archive and compress files `rsync -az`

[the remark this answers is not in the document]
Comment (artemiy312) [reply etc/backup/backup-entrypoint.sh]: @duker33 , 
>store backups on the same machine is not good idea

I totally agree with it, so we will get an additional server for it?

[the remark this answers is not in the document]
Comment (duker33) [reply etc/backup/backup-entrypoint.sh]: @artemiy312 , y, we already have one with lot's of hdd

File: etc/backup/backup-restore.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/env sh
+
+# We'll rewrite restore according for new rsync based backup system.
+# See `backup-entrypoint.sh`
+
+# @todo #232 - DB restore error on local env.
```
Comment (artemiy312) [inline etc/backup/backup-restore.sh]: It seems this todo isn't relevant for new the backup system, so we may drop it

**Pull request merged**